package.json

```diff
@@ -61,5 +61,9 @@
     "**/*.js": [
       "prettier --write"
     ]
+  },
+  "dependencies": {
+    "@types/minimist": "^1.2.2",
+    "minimist": "^1.2.7"
```

Comment:
Please move these to `devDependencies`, otherwise they get installed to all users of the rp2040js library

Reply:
moved to `devDependencies`